### PR TITLE
Create News API.py

### PR DESCRIPTION
**Pull request by SachinDabhade**
# Pull Request Template

**News API in python**

**Changes made:**<br>
mention your changes here

### Self Check(Tick After Making pull Request)

- [x] I have read and followed the [Contribution Guidelines](/CONTRIBUTING.md)
- [x] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in the proper folder

- [x] PR is regarding Code
- [x] I am following clean code and Documentation and my code is well linted with flake8.
- [ ] I am using external libraries and I have created requirements.txt in my project folder
- [x] Script is tested and running perfectly fine on my system

- [ ] PR is regarding the documentation

> If the issue was not assigned to you Please don't make a PR, It will be marked as invalid.

Discuss in discord server:- http://pragmaticprogrammer.in/discord

## acknowledgment
- [x] I have tested my code and distributing it under GPL-3.0 License

### CODE REVIEW

**Review by Punit-Choudhary — CHANGES_REQUESTED** [review]
The file is not in the proper folder, please go through [CONTRIBUTE](https://github.com/Punit-Choudhary/Python-beginner-scripts/tree/main/CONTRIBUTE)!

**Comment by SachinDabhade** [comment]
I have deleted the previous file and inserted a new file, please give me a review on it if anything is wrong

**Review by Punit-Choudhary — CHANGES_REQUESTED** [review]
Please create `requirements.txt` for third-party libraries

**Comment by SachinDabhade** [comment]
There is no requirements.txt is needed

**Comment by Punit-Choudhary** [comment]
> please review this pull request...

You are using two third-party libraries:
- `wincom32` which is inside `Pywin32`
- 'requests`

Please make a folder for your project and add these requirements in `requirements.txt`

**Comment by SachinDabhade** [comment]
Added Requirements.txt in appropriate directory with program

File: Projects/NEWS API/NEWS API.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+        # Checking the input category is valid or not
+        if category.lower() not in categories:
+            print("Invalid input....! Please try again....!")
+            speak("Invalid input....! Please try again....!")
+            exit()
+
+        # url from which the news will fetch
+        url = (
+            f'https://newsapi.org/v2/top-headlines?country={country1.lower()}&category={category.lower()}&apiKey=2114b271bda54165b23a311b1d1e3c49')
```
Comment (Punit-Choudhary) [inline Projects/NEWS API/NEWS API.py:74]: API key exposed, please use `.env` to store credentials.
Refer to this blog to learn more about the environment variables 😇 
https://dev.to/jakewitcher/using-env-files-for-environment-variables-in-python-applications-55a1

**Review by Punit-Choudhary — CHANGES_REQUESTED** [review]

**Comment by SachinDabhade** [comment]
Whta if I didn't add API key

**Pull request merged**